ros2_control_test_assets/include/ros2_control_test_assets/descriptions.hpp

```diff
@@ -26,13 +26,39 @@ namespace ros2_control_test_assets
 const auto urdf_head =
   R"(
 <?xml version="1.0" encoding="utf-8"?>
+<!-- =================================================================================== -->
+<!-- |    This document was autogenerated by xacro from minimal_robot.urdf.xacro       | -->
+<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
+<!-- =================================================================================== -->
 <robot name="MinimalRobot">
+  <!--  <xacro:include filename="$(find ros2_control_demo_robot)/description/demo_2dof/robot_2dof.urdf.xacro" />
+  <xacro:include filename="$(find ros2_control_demo_robot)/description/demo_2dof/robot_2dof.gazebo.xacro" />
+  <xacro:include filename="$(find ros2_control_demo_robot)/description/demo_2dof/robot_2dof.ros2_control.xacro" />-->
```

Comment:
that's introducing an exec dependency to `ros2_control_demo_robot`. I don't think we want that here - or `xacro` to that end.

Reply:
Removing this, because is commented out.